File: dev-docs/snippets/auth/is-authenticated.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Check whether the client currently holds a valid auth token.
export function isAuthenticated(client: JsBaoClient) {
  // #region example
  const signedIn = client.isAuthenticated();
  // #endregion example
  return signedIn;
}

File: dev-docs/snippets/auth/is-authenticated.swift
import JsBaoClient

// Check whether the client currently holds a valid auth token via the typed
// `client.auth` namespace (synchronous, local).
func isAuthenticated(client: JsBaoClient) {
  // #region example
  let signedIn = client.auth.isAuthenticated()
  // #endregion example
  _ = signedIn
}
